app/webhook.go

```diff
@@ -440,6 +473,12 @@ func (a *App) CreateOutgoingWebhook(hook *model.OutgoingWebhook) (*model.Outgoin
 		}
 	}
 
+	if hook.Signed {
+		hook.SecretToken = model.NewRandomString(26)
+	} else {
+		hook.SecretToken = ""
```

Comment:
given that zero values for strings default to "", it it necessary to be explicit here? https://go.dev/ref/spec#The_zero_value

Reply:
@mxschumacher, thanks for the feedback. It's been a while and not 100% sure why not signed hooks should be explicitly set the empty string. Maybe there is a case that SecretToken carries an old value that needs to clear out. This request has been hanging there too long, not sure that MM still needs a such feature.